src/main/java/jenkins/plugins/openstack/compute/OpenStackDSL.java

```diff
@@ -0,0 +1,26 @@
+package jenkins.plugins.openstack.compute;
+
+import groovy.lang.Binding;
+import hudson.Extension;
+import org.jenkinsci.plugins.workflow.cps.CpsScript;
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable;
+
+@Extension public class OpenStackDSL extends GlobalVariable {
+
+    @Override public String getName() {
+        return "openstack";
+    }
+
+    @Override public Object getValue(CpsScript script) throws Exception {
+        Binding binding = script.getBinding();
+        Object openstack;
+        if (binding.hasVariable(getName())) {
+            openstack = binding.getVariable(getName());
+        } else {
+            openstack = script.getClass().getClassLoader().loadClass("jenkins.plugins.openstack.compute.OpenStack").getConstructor(CpsScript.class).newInstance(script);
```

Comment:
Does the good old `new jenkins.plugins.openstack.compute.OpenStack` not work as expected?